WebRoot/js/zimbraAdmin/common/ZaTree.js

```diff
@@ -525,17 +535,16 @@ function(item, ev) {
         this._selectedItems.add(item);
 
         if (item._setSelected(true)) {
-            if ((item._text === ZaMsg.NAD_Tab_HSM && typeof ZaHSM !== 'undefined' && ZaHSM.checkStorageFeatureEnabled()) ||
-                item._text !== ZaMsg.NAD_Tab_HSM) {
+            if (!isLicensedItem || isLicenseValid) {
```

Comment:
after removing the above lines of code are we able to achieve the existing behaviour in Storage management?

Reply:
I have made changes accordingly in Storage management admin extension and raised a PR.

PR link: https://github.com/Zimbra/zm-hsm-admin-zimlet/pull/65